game.py

```diff
@@ -252,11 +250,11 @@ def _test_game_over(self, new_dot):
            self._dots[
             new_dot + CIRCLE[c][4][0] + THIRTEEN * CIRCLE[c][4][1]].type and \
            self._dots[
-            new_dot + CIRCLE[c][5][0] + THIRTEEN * CIRCLE[c][5][1]].type:
-           # Game-over feedback
-           for dot in self._dots:
-               dot.set_label(':)')
-           return True
+                new_dot + CIRCLE[c][5][0] + THIRTEEN * CIRCLE[c][5][1]].type:
```

Comment:
It would aid comprehension if this line matched the indentation of the other five lines above that start with new_dot, but I did not find a way to do that.